cli/EdgeDbGenerator.res

```diff
@@ -341,6 +350,20 @@ module AnalyzeQuery = {
           distinctTypes,
         },
       )->generateSetType(cardinality)
+      if (
+        distinctTypes->Set.size === 0 ||
+          !(
+            distinctTypes
+            ->Set.values
+            ->Iterator.toArrayWithMapper(t => String.includes(t, "response"))
+            ->Array.includes(true)
+          )
+      ) {
```

Comment:
Could you add a little comment about the logic here and why it is like it is?

Reply:
How does that sound?